manifests/l/LargeAddressAware/laa/2.0.4/LargeAddressAware.laa.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: LargeAddressAware.laa
+PackageVersion: 2.0.4
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Large Address Aware.exe
+  PortableCommandAlias: largeaddressaware
+ArchiveBinariesDependOnPath: true
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.techpowerup.com/forums/attachments/laa_2_0_4-zip.34392/

```

Comment:
This doesn't seem like an official distribution channel, as it's just a forum attachment. Is this software officially published anywhere?

Reply:
No, it isn't.